qiskit_experiments/test/t1_backend.py

```diff
@@ -0,0 +1,138 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+A simulator for T1 experiment for testing and documentation
+"""
+
+import numpy as np
+from qiskit.providers.backend import BackendV1
+from qiskit.providers.options import Options
+from qiskit.providers.models import QasmBackendConfiguration
+from qiskit.result import Result
+from qiskit_experiments.test.mock_job import MockJob
+
+
+class T1Backend(BackendV1):
```

Comment:
Can this be more simplified by using configured fake backends? See IQ backend for example.

Reply:
Looking at the fake backends in Terra (directory `qiskit/test/mock`), they are too heavy and not really correct for this case (for example they have the CX gate in the properties). Anyway the T1 backend is hopefully temporary, to be replaced by Ethan's general backend.

Reply:
Okey, then this is fine.